Pdf/Helper/Data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace MagentoJapan\Pdf\Helper;
+
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\App\Helper\Context;
+
+/**
+ * Class Data
+ * @package MagentoJapan\Pdf\Helper
+ */
+class Data extends \Magento\Framework\App\Helper\AbstractHelper
+{
+
+    /**
+     * font file path
+     */
+    const FONT_DIR = 'lib/fonts/';
+
+    /**
+     * @var string
+     */
+    protected $_dir;
```

Comment:
Current edition of [Magento Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html) suggests to use composition over inheritance, therefore it may be better to change modifier to `private`.